stix_shifter_modules/aws_athena/stix_translation/json/stix_2_1/ocsf_from_stix_map.json

```diff
@@ -0,0 +1,820 @@
+        "src_endpoint.name"
+      ]
+    }
+  },
+  "x-oca-event": {
+    "fields": {
+      "action": [
+        "activity",
+        "activity_name"
+      ],
+      "category": [
+        "category_name"
+      ],
+      "code": [
+        "activity_id",
+        "category_uid"
+      ],
+      "confidence": [
+        "confidence"
+      ],
+      "created": [
+        "time"
+      ],
+      "duration": [
+        "duration"
+      ],
+      "extensions.'x-cloud-api'.class_uid": [
+        "class_uid"
+      ],
+      "module": [
+        "class_name"
+      ],
+      "network_ref.dst_ref.value": [
+        "dst_endpoint.ip"
+      ],
+      "network_ref.src_ref.value": [
+        "dst_endpoint.ip"
+      ],
+      "timezone": [
+        "timezone_offset"
+      ]
+    }
+  },
+  "x-ocsf-cloud": {
+    "fields": {
+      "account_type": [
+        "cloud.account_type"
+      ],
+      "account_type_id": [
+        "cloud.account_type_id"
+      ],
+      "account_uid": [
+        "cloud.account_uid"
+      ],
+      "api_version": [
+        "api.version"
+      ],
+      "message": [
+        "message"
+      ],
+      "operation": [
+        "api.operation"
+      ],
+      "org_uid": [
+        "cloud.org_uid"
+      ],
+      "profiles": [
+        "profiles"
+      ],
+      "project_uid": [
+        "cloud.project_uid"
+      ],
+      "provider": [
+        "cloud.provider"
+      ],
+      "raw_data": [
+        "raw_data"
+      ],
+      "ref_event_code": [
+        "ref_event_code"
+      ],
+      "ref_event_name": [
+        "ref_event_name"
+      ],
+      "ref_event_uid": [
+        "ref_event_uid"
+      ],
+      "ref_time": [
+        "ref_time"
+      ],
+      "region": [
+        "cloud.region"
+      ],
+      "request_flags": [
+        "api.request.flags"
+      ],
+      "request_uid": [
+        "api.request.uid"
+      ],
+      "resource_uid": [
+        "cloud.resource_uid"
+      ],
+      "response_code": [
+        "api.response.code"
+      ],
+      "response_error": [
+        "api.response.error"
+      ],
+      "response_error_message": [
+        "api.response.error_message"
+      ],
+      "response_flags": [
+        "api.response.flags"
+      ],
+      "response_message": [
+        "api.response.message"
+      ],
+      "service_labels": [
+        "api.service.labels"
+      ],
+      "service_name": [
+        "api.service.name"
+      ],
+      "service_uid": [
+        "api.service.uid",
+        "api.service.version"
+      ],
+      "severity": [
+        "severity"
+      ],
+      "status": [
+        "status"
+      ],
+      "status_code": [
+        "status_code"
+      ],
+      "status_detail": [
+        "status_detail"
+      ],
+      "status_id": [
+        "status_id"
+      ],
+      "type_name": [
+        "type_name"
+      ],
+      "type_uid": [
+        "type_uid"
+      ],
+      "zone": [
+        "cloud.zone"
+      ]
+    }
+  },
+  "x-ibm-ttp-tagging": {
```

Comment:
If you are introducing a common custom STIX object, you should also raise a PR in the stix-extensions project with it's definition https://github.com/opencybersecurityalliance/stix-extensions

Reply:
its actually already there. https://github.com/opencybersecurityalliance/stix-extensions/blob/main/2.0/x-ibm-ttp-tagging.md